### PR TITLE
Test against older Rubies / Rails

### PR DESCRIPTION
**Pull request by greysteil**
That.

### CODE REVIEW

**Comment by greysteil** [comment]
@hmarr - what's not to like, right?

File: Gemfile [hunk 1]
```diff
@@ -2,6 +2,8 @@ source "https://rubygems.org"
 
 gemspec
 
+gem "rails", "~> #{ENV['RAILS_VERSION']}" if ENV["RAILS_VERSION"]
```
Comment (hmarr) [inline Gemfile]: Swap out rails for just activerecord?

Comment (hmarr) [inline Gemfile]: And you should be able to use double quotes both places. Y'know for consistency :)

**Comment by greysteil** [comment]
@hmarr - updated. Rubocop doesn't like double quotes within string interpolation, though...

**Comment by hmarr** [comment]
Love it :+1:

**Comment by greysteil** [comment]
@hmarr - could you cut a patch release with this in it, please? The last release still insists on Rails <= 4.2, and is blocking us testing against 5.0.0.rc1.

**Comment by hmarr** [comment]
@greysteil just pushed out v1.0.0

**Pull request merged**